install.rb

```diff
@@ -204,16 +180,6 @@ def prepare_installation
     opts.on('--no-batch-files', 'Prevents installation of batch files for windows', 'Default off') do |batch_files|
       InstallOptions.batch_files = false
     end
-    opts.on('--quick', 'Performs a quick installation. Only the', 'installation is done.') do |quick|
-      InstallOptions.rdoc    = false
-      InstallOptions.ri      = false
-      InstallOptions.configs = true
-    end
```

Comment:
We use `--quick` when building puppet-agent packages. Here's where that happens https://github.com/puppetlabs/puppet-agent/blob/b7976e5c12a8816f00d48a45266ab0f27ebef379/configs/components/puppet.rb#L145-L161

Reply:
Ok, but it does nothing now. Do you prefer to keep it in as a dummy option, raising a warning?

Reply:
yes a noop is fine with a warning to stderr

Reply:
I kept in `configs = true` since technically that does something, even though it's on by default already.